packages/asc-ui/src/Radio/Radio.tsx

```diff
@@ -1,8 +1,13 @@
 import * as React from 'react'
-import { Radio as RadioStyled } from '@amsterdam/asc-core'
-
+import { Radio as RadioStyled, RadioLabel, RadioItem } from '@amsterdam/asc-core'
+console.log(RadioItem);
 const Radio = (props: any) => (
```

Comment:
I think we should use props here, then it would be clear for the user (IDE's etc.) which props he can pass

Reply:
Yes exactly, but also state the prop types in typescript :)

Reply:
I refactored that